.env

```diff
@@ -0,0 +1,6 @@
+NEXT_PUBLIC_API_KEY="AIzaSyDkQP_5mU8RLDIBUdhVcYRgmtUfm1OZovs"
```

Comment:
You pushed .env file.
This one we need to remove. And probably we would need to change all keys...we need to check:)